modules/test/playwright/tests/object-web/objectActions.spec.ts

```diff
@@ -71,17 +85,185 @@ test.describe('manage object actions through object actions tab', () => {
 					.getByRole('option', {name: names[index]})
 			).toBeVisible();
 		}
+	});
+});
 
-		// Clean up
+test('can send notification email via download action', async ({
+	apiHelpers,
+	page,
+	viewObjectEntriesPage,
+}) => {
 
-		await apiHelpers.objectAdmin.deleteObjectDefinition(
-			objectDefinition.id
+	// Create email notification template
+
+	const notificationTemplate =
+		await apiHelpers.notification.postRandomNotificationTemplate(
+			'notification template test ' + getRandomInt()
 		);
 
-		for (let index = 0; index < ids.length; index++) {
+	createdEntities.notificationTemplateIds = [notificationTemplate.id];
+
+	// Create object definition with an attachment field
+
+	const objectDefinition = await apiHelpers.objectAdmin.postObjectDefinition({
+		active: true,
+		externalReferenceCode: 'customObjectERC',
+		label: {
+			en_US: 'customobject',
+		},
+		name: 'CustomObject',
+		objectFields: [
```

Comment:
Hey! I think that this is too much information to be inside the test file. If every test that needs a specific definition define the definition inside the test file, then we will have a huge file that is hard to read. 

Please, create a .ts file inside the dependencies folder and declare and export an object definition inside that file where every key is an object field, like:
```
export const mockedObjectFields = {
       attchmentField = 	{
		DBType: 'Long',
		businessType: 'Attachment',
		externalReferenceCode: 'testAttachment',
		indexed: true,
		indexedAsKeyword: false,
		indexedLanguageId: '',
		label: {
			en_US: 'testAttachment',
		},
		listTypeDefinitionId: 0,
		name: 'testAttachment',
		objectFieldSettings: [
			{
				name: 'acceptedFileExtensions',
				value: 'jpeg, jpg, pdf, png, txt',
			},
			{
				name: 'maximumFileSize',
				value: 100,
			},
			{
				name: 'fileSource',
				value: 'userComputer',
			},
			{
				name: 'showFilesInDocumentsAndMedia',
				value: false,
			},
		],
		required: false,
		state: false,
		system: false,
		type: 'Long',
		unique: false,
	},
}
```
 Then import to this file and assign to `objectFields`:

`objectFields: [mockedObjectFields.attchmentField]`

Reply:
Also, try to use `postRandomObjectDefinition()` for a even more concise object creation.

Reply:
Done!